.github/workflows/ci-update-workflow.yml

```diff
@@ -0,0 +1,62 @@
+name: ci-update-workflow
+
+on:
+  schedule:
+  # Run every hour.
+  - cron:  '0 * * * *'
```

Comment:
Lets run this less frequently, like every month. Not sure how frequently go versions are made available.